### PR TITLE
Playbook site

### PR DESCRIPTION
**Pull request by mrfussion**
Integrating the site playbook and role-specific micro-playbooks.

### CODE REVIEW

File: playbooks/tinc.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Install and configure VPN.
+---
+- name: Install and configure VPN
+  hosts: all
+  roles:
+    - role: tina_pm.third_party.tinc
+      when: inventory_hostname in groups['tinc_servers']
+      tags: tinc
```
Comment (NightTsarina) [inline playbooks/tinc.yaml:10]: This was a hack to run the role in the same play that targetted all hosts, but here you can just change the `hosts` value
```suggestion
  hosts: tinc_servers
  roles:
    - role: tina_pm.third_party.tinc
      tags: tinc
```

**Review by NightTsarina — APPROVED** [review]
LGTM, thanks!!

**Review by dato — APPROVED** [review]
LGTM!

File: playbooks/apache2.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Setup apache2 server and configurations.
+---
+- name: Setup apache2 server and configurations
+  hosts: all
```
Comment (dato) [inline playbooks/apache2.yaml:6]:
```suggestion
  hosts: apache_servers
```

Thoughts @NightTsarina? (Also for other playbooks below.)

**Pull request merged**